mmdet/models/backbones/common.py

```diff
@@ -0,0 +1,1649 @@
+"""
```

Comment:
why do you bring these files here? face-detection-010* models don't require them? Why? Are there some modifications?